onnxruntime/test/onnx/main.cc

```diff
@@ -637,6 +637,66 @@ select from 'TF8', 'TF16', 'UINT8', 'FLOAT', 'ITENSOR'. \n)");
             ORT_TSTR("bernoulli_double"),
             ORT_TSTR("bernoulli_seed")};
 
+    // bfloat16 is not a standard type.
+    // rounding when casting to string is not constant accross platforms.
+    static const ORTCHAR_T* blfoat16_or_string_cast_tests[] = {
+        ORT_TSTR("cast_BFLOAT16_to_FLOAT"),
+        ORT_TSTR("cast_FLOAT_to_BFLOAT16"),
+        ORT_TSTR("cast_FLOAT_to_STRING"),
+        ORT_TSTR("castlike_BFLOAT16_to_FLOAT"),
+        ORT_TSTR("castlike_BFLOAT16_to_FLOAT_expanded"),
+        ORT_TSTR("castlike_FLOAT_to_STRING"),
+        ORT_TSTR("castlike_FLOAT_to_STRING_expanded")};
```

Comment:
maybe just leave these ones as they were in `broken_tests` below? I feel that it had some useful per-test reason strings before. plus I can't think of a better name for this list...

Reply:
restored to its original configuration